src/SprykerEco/Zed/Unzer/Persistence/UnzerRepository.php

```diff
@@ -228,7 +228,7 @@ public function getStoreRelationByIdUnzerCredentials(int $idUnzerCredentials): S
     public function findUnzerCredentialsCollectionByCriteria(
         UnzerCredentialsCriteriaTransfer $unzerCredentialsCriteriaTransfer
     ): UnzerCredentialsCollectionTransfer {
-        $unzerCredentialsQuery = $this->getFactory()->createUnzerCredentialsQuery();
+        $unzerCredentialsQuery = $this->getFactory()->createUnzerCredentialsQuery()->joinWithUnzerCredentialsStore();
         $unzerCredentialsQuery = $this->setUnzerConfigFilters(
```

Comment:
as I understand table from `Store` module is used inside of `setUnzerConfigFilters`
please add `@module` annotation for this method respectively